README.md

```diff
@@ -58,30 +59,23 @@ These features are brought by [ftpserverlib](https://github.com/fclairamb/ftpser
 #### Download it
 Fetch a binary from the [latest release](/releases) and run it.
 
-#### Build it
+#### Build & run it locally
 
 ```bash
 go install github.com/fclairamb/ftpserver@main
+
+ftpserver &
 ```
 
 #### Run it with docker
 There's also a containerized version of the server (31MB, based on alpine).
 
 ```sh
-# Starting the sample FTP server
-docker run --rm -d -p 2121-2130:2121-2130 -v ./ftpserver/files:/tmp -v ./ftpserver:/app fclairamb/ftpserver
-
-# Download some file
-[ -f kitty.jpg ] || (curl -o kitty.jpg.tmp https://placekitten.com/2048/2048 && mv kitty.jpg.tmp kitty.jpg)
-
-# Upload it
-curl -v -T kitty.jpg ftp://test:test@localhost:2121/
+# Creating a directory
+mkdir -p files
 
-# Download it back
-curl ftp://test:test@localhost:2121/kitty.jpg -o kitty2.jpg
-
-# Compare it
-diff kitty.jpg kitty2.jpg
+# Starting the sample FTP server
+docker run --rm -d -p 2121-2130:2121-2130 -v $(pwd):/app fclairamb/ftpserver
```

Comment:
Any reason for not including the `/tmp` path to the `files` directory you created above.
To make the data persistent
```
docker run --rm -d -p 2121-2130:2121-2130 -v $(pwd)/files:/tmp -v $(pwd):/app fclairamb/ftpserver
```
Am I missing something in the documentation ?

Reply:
Well, I hesitated and left the `files` directory and dropped the -v $(pwd)/files:/tmp... Because `/tmp` is not the smartest path for persistence 😬 and I would like to keep the local & docker config common for the demo.

But I guess I can keep it that way.

Reply:
Yes you are right, `/tmp` for once is quite ambiguous in certain sense.